### ISSUE TITLE
case sensitivity bug with multiselect

### ISSUE THREAD

**Issue by thanhvuong** [furqanZafar/react-selectize#20]
When there are 2 items in multiple select with the same label but with different case, selecting one will remove both in multiselect.

I have 2 items in the dropdown:

```
<MultiSelectBox options={[{label: "element type 1", value: "3899"},{label: "Element type 1",value: "3872"}]} label="Element Types" placeholder="Select Element Types" value="" field="elementTypes" formName="createView" />

```

**Comment by furqanZafar** [comment]
fixed in v0.3.11

**Comment by babsonmatt** [comment]
Thanks for the quick fix!
